app/views/users/twitter_search_user.html.erb

```diff
@@ -0,0 +1,27 @@
+<h1><%= @user_name %></h1>
+<%= image_tag(@user_tweets[0].user.profile_image_url) %>
+<br>
+<br>
```

Comment:
Try to avoid using line breaks for styling purposes. It's best to adjust margin or padding in CSS.